src/com/twu/biblioteca/MainMenu.java

```diff
@@ -9,7 +9,9 @@ public void welcome() {
     }
 
     public void showOptions() {
-        System.out.println("Please select an option from the following:\n" + "1. List of books");
```

Comment:
I like this natural refactor to separate print lines once a second option was needed